lib/src/main/java/ECSE428Project/service/AccountService.java

```diff
@@ -35,6 +36,29 @@ public Account createAccount(String name, String email, String password) {
         account = accountRepository.save(account);
         return account;
     }
+    
+  @Transactional
+  public Account changePassword(String id, String oldPass, String newPass) {
```

Comment:
the "id" is the email address right?

Reply:
yes it is, should I change the variable name?